core/src/serve_repair.rs

```diff
@@ -360,24 +361,26 @@ impl ServeRepair {
             inc_new_counter_debug!("serve_repair-handle-repair--eq", stats.self_repair);
         }
 
-        inc_new_counter_info!("serve_repair-total_packets", stats.total_packets);
-        inc_new_counter_info!("serve_repair-dropped_packets", stats.dropped_packets);
-
-        debug!(
-            "repair_listener: total_packets: {} passed: {}",
-            stats.total_packets, stats.processed
+        datapoint_info!(
+            "serve_repair-requests_received",
+            ("total_requests", stats.total_requests, i64),
+            ("dropped_requests", stats.dropped_requests, i64),
+            ("total_response_packets", stats.total_response_packets, i64),
+            ("self-repair", stats.self_repair, i64),
```

Comment:
`self_repair` instead of `self-repair` for consistency end easier queries.
(`self-repair` would require quoting in queries)